doc/plugin_server_nodeattestor_k8s_psat.md

```diff
@@ -20,6 +20,7 @@ The main configuration accepts the following values:
 | Configuration | Description                                                                       | Default |
 |---------------|-----------------------------------------------------------------------------------|---------|
 | `clusters`    | A map of clusters, keyed by an arbitrary ID, that are authorized for attestation. |         |
+|               | May be empty, when no clusters are authorized for attestation.                    |         |
```

Comment:
This renders as a separate row in the markdown and looks a little funny. I feel like we should just add this to the end of the sentence in the description above.

Reply:
Altered to be a warning note in the next push.